Sources/TuistDependencies/SwiftPackageManager/Utils/SwiftPackageManagerGraphGenerator.swift

```diff
@@ -85,7 +85,7 @@ public final class SwiftPackageManagerGraphGenerator: SwiftPackageManagerGraphGe
             case "remote":
                 packageFolder = checkoutsFolder.appending(component: dependency.subpath)
             case "local":
-                guard let path = dependency.packageRef.path else {
+                guard let path = dependency.packageRef.path ?? dependency.packageRef.location else {
```

Comment:
maybe you could leave a message why `path` fallbacks to `location` 

it is not obvious on the first glance

Reply:
you could consider creating a new computable variable inside `packageRef` that implements this fallback

Reply:
I would keep that type just as a model (as the PackageInfo), and keep the logic here